src/frontend/js/components/SaleTunnel/SaleTunnelInformation/index.tsx

```diff
@@ -0,0 +1,65 @@
+import { Alert, Input, VariantType } from '@openfun/cunningham-react';
+import { defineMessages, FormattedMessage, useIntl } from 'react-intl';
+import { AddressSelector } from 'components/SaleTunnel/AddressSelector';
+import { CreditCardSelector } from 'components/SaleTunnel/CreditCardSelector';
+import { PaymentScheduleGrid } from 'components/PaymentScheduleGrid';
+
+const messages = defineMessages({
+  title: {
+    id: 'components.SaleTunnel.Information.title',
+    description: 'Title for the information section',
+    defaultMessage: 'Information',
+  },
+  description: {
+    id: 'components.SaleTunnel.Information.description',
+    description: 'Description of the information section',
+    defaultMessage: 'Those information will be used for billing',
+  },
+  fullNameLabel: {
+    id: 'components.SaleTunnel.Information.fullNameLabel',
+    description: 'Label for the full name input',
+    defaultMessage: 'Full name',
+  },
+});
+
+export const SaleTunnelInformation = () => {
+  const intl = useIntl();
+  return (
+    <div className="sale-tunnel__information">
+      <div>
+        <h3 className="block-title mb-t">
+          <FormattedMessage {...messages.title} />
+        </h3>
+        <div className="description mb-s">
+          <FormattedMessage {...messages.description} />
+        </div>
+        <Input label={intl.formatMessage(messages.fullNameLabel)} fullWidth={true} />
+        <AddressSelector />
```

Comment:
When the user have no addresses, we could display "No billing address" and disable the select.

Reply:
I like the idea, but at the moment Cunningham does not provide this feature :/